12-filemake.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "Hello, let's make a file!"
+echo "What is the file name?"
+ read filename
+
+touch $filename
+
+echo "done!"
+
+
```

Comment:
Your Git configuration seems to be broken @plourday . Most likely your email address doesn't match the one on Github.